tools/ci_build/github/azure-pipelines/linux-cpu-minimal-build-ci-pipeline.yml

```diff
@@ -0,0 +1,67 @@
+jobs:
+- job: Linux_CPU_Minimal_Build
+  timeoutInMinutes: 120
+  workspace:
+    clean: all
+  pool: Linux-CPU
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - task: Docker@2
+    displayName: login
+    inputs:
+      containerRegistry: onnxruntimeregistry
+      command: login
+      addPipelineData: false
+  - task: CmdLine@2
+    inputs:
+      script: |
+        mkdir -p $HOME/.models
+        cp $(Build.SourcesDirectory)/onnxruntime/test/testdata/ort_github_issue_4031.onnx $HOME/.models/
+        mkdir -p $HOME/.onnx
```

Comment:
Please document the reasons things like this in comments so it's easier to maintain the build config.

e.g. we copy this from testdata to a separate folder due to the --include_ops_by_model recursively searching the provided folder for all onnx models, and we only want to include ops for this model in the build. 

Would be good to capture somewhere that there's a matching .ort file for this model, and that a unit test validates the build using that model (which explains why we copy this particular model and why it matters).